post.md

```diff
@@ -1,22 +1,22 @@
 For a while I've wanted to explore Suspense.
```

Comment:
Question 1? Who is the intended audience? I assume technical and React-y, but how beginner? 
Question 2? What is the goal of the piece? To document your journey with Suspense? To teach us how to use Suspense? To warn us off Suspense? 
Question 3? What prerequisites do I need for this?